perun-core/src/main/java/cz/metacentrum/perun/core/entry/VosManagerEntry.java

```diff
@@ -694,6 +696,23 @@ public BanOnVo setBan(PerunSession sess, BanOnVo ban) throws PrivilegeException,
 		return vosManagerBl.setBan(sess, ban);
 	}
 
+	@Override
+	public BanOnVo updateBan(PerunSession sess, BanOnVo banOnVo) throws PrivilegeException, MemberNotExistsException, BanNotExistsException, VoNotExistsException {
+		Utils.checkPerunSession(sess);
+		vosManagerBl.getBanById(sess, banOnVo.getId());
+		Member member = getPerunBl().getMembersManagerBl().getMemberById(sess, banOnVo.getMemberId());
+		Vo vo = getPerunBl().getVosManagerBl().getVoById(sess, banOnVo.getVoId());
+
+		// Authorization
+		if (!AuthzResolver.authorizedInternal(sess, "updateBan_BanOnVo_policy", Arrays.asList(banOnVo, member, vo))) {
```

Comment:
```suggestion
		if (!AuthzResolver.authorizedInternal(sess, "updateBan_BanOnVo_policy", Arrays.asList(banOnVo))) {
```
Vo and Member get fetched from BanOnVo, no need to include them here